esphome/components/bl0940/button/calibration_reset_button.cpp

```diff
@@ -0,0 +1,22 @@
+#include "calibration_reset_button.h"
+#include "../bl0940.h"
+#include "esphome/core/hal.h"
+#include "esphome/core/log.h"
+#include "esphome/core/application.h"
+
+namespace esphome {
+namespace bl0940 {
+
+static const char *const TAG = "calibration_reset.button";
```

Comment:
Tag should always start with the component name
```suggestion
static const char *const TAG = "bl0940.button.calibration_reset";
```